### PR TITLE
Addons: remove old `X-RTD-Hosting-Integrations` HTTP header

### PR DESCRIPTION
**Pull request by humitos**
Remove old HTTP header used by `build.commands`. All the projects are using the new addons now, so this header is not used anymore.

### CODE REVIEW

**Review by stsewd — APPROVED** [review]
Anything else we need here? Tests are failing bc of this.

**Comment by humitos** [comment]
We can move the removal of that particular test to another PR if we want, so we can merge that removal quicker.

**Comment by humitos** [comment]
I'm adding this PR to the next sprint since we are all already in addons. Besides updating this PR, we will want to remove the this header from the CF worker as well.

File: dockerfiles/nginx/proxito.conf.template [hunk 1]
```diff
@@ -122,7 +122,6 @@ server {
         # This header allows us to decide whether or not inject the script at CloudFlare level
         # Now, I'm injecting it in all the NGINX responses because `sub_filter` is not allowed inside an `if` statement.
         set $rtd_hosting_integrations $upstream_http_x_rtd_hosting_integrations;
```
Comment (ericholscher) [inline dockerfiles/nginx/proxito.conf.template:124]:
```suggestion
```

Need to delete both I think?

**Review by ericholscher — APPROVED** [review]

File: dockerfiles/nginx/proxito.conf.template (hunk 1 above)
Comment (humitos) [reply dockerfiles/nginx/proxito.conf.template:124]: Yes. Missed that.

**Pull request merged**